erpnext/projects/doctype/task/task.py

```diff
@@ -12,6 +12,7 @@
 
 
 class CircularReferenceError(frappe.ValidationError): pass
+class EndDateConnotGreaterThanProjecteEndDateError(frappe.ValidationError): pass
```

Comment:
A couple of typos here:

- `Connot` -> `Cannot`
- `EndDateConnotGreater` -> `EndDateCannotBeGreater`
- `ProjecteEndDate` -> `ProjectEndDate`

Reply:
Done